### ISSUE TITLE
Add a `--confirm` option which would ask the user for confirmation before sending a mutating request

### ISSUE THREAD

**Issue by gr2m** [octoherd/cli#2]
Example

```
$ npx mutate-github-repositories \
  --token 0123456789012345678901234567890123456789 \
  example.js \
  gr2m/mutate-github-repositories-cli \
  --confirm
```

Would prompt the user with something like this

```
? Do you want to allow the following request
  PUT /user/starred/gr2m/mutate-github-repositories-cli
> Yes
  No 
```

Bonus points to add a reply option that would allow to confirm for all future requests to the same route (In this case [`PUT /user/starred/{owner}/{repo}`](https://docs.github.com/en/rest/reference/activity#star-a-repository-for-the-authenticated-user)

Triple bonus points if we could show some meta information about the current request using GitHub's OpenAPI spec: https://github.com/github/rest-api-description/

Altogether this could look like this

```
? Do you want to allow the following request:
  Star a repository for the authenticated user
  route: PUT /user/starred/{owner}/{repo}
  parameters: 
    owner: gr2m
    repo: mutate-github-repositories-cli
  documentation: https://docs.github.com/en/rest/reference/activity#star-a-repository-for-the-authenticated-user

> Yes
  No
  Allow all "PUT /user/starred/{owner}/{repo}" requests
```

**Comment by gr2m** [comment]
We do have `----octoherd-bypass-confirms` now, and ask for confirmation by default. When asking for confirmation, it's a yes/no question. I would prefer a multi-choice question instead, like the example above shows

**Comment by gr2m** [comment]
released in v3.1
